spec/models/digest_run_spec.rb

```diff
@@ -31,20 +31,20 @@
 
       context "weekly" do
         it "sets starts_at to 8am on date - 1.week" do
-          date = 1.day.ago
-          instance = described_class.create!(date: date, range: "weekly")
+          saturday = Date.new(2020, 9, 5)
+          instance = described_class.create!(date: saturday, range: "weekly")
 
           expect(instance.starts_at).to eq(
-            Time.zone.parse("08:00", (date - 1.week)),
+            Time.zone.parse("08:00", (saturday - 1.week)),
           )
         end
 
         it "sets ends_at to 8am on date" do
-          date = Date.current
-          instance = described_class.create!(date: date, range: "weekly")
+          saturday = Date.new(2020, 9, 5)
```

Comment:
It looks like this fixes a flaky test I saw pop up this morning where it seems our test suite fails if it [runs before 8am](https://ci.integration.publishing.service.gov.uk/job/email-alert-api/job/dependabot%252Fbundler%252Frails-6.0.3.3/1/console) 🤦  So that's a nice bonus